src/components/ToTop/styles.less

```diff
@@ -0,0 +1,22 @@
+.scroll-to-top {
+  cursor: pointer;
+  height: 50px;
+  width: 50px;
+  text-align: center;
+  border-radius: 50%;
+  box-shadow: 0 0 3px #0182F033;
```

Comment:
here

Reply:
It will be improved in the next version.